targets_test.go

```diff
@@ -36,3 +36,21 @@ func TestOKKubeConform(t *testing.T) {
 		t.Fatalf(`kubeConform(paths,"api-platform) should pass but failed with error %v`, err)
 	}
 }
+
+// Test for manifest files expected to fail Kyverno policy validation
+// func TestFailedValidateKyverno(t *testing.T) {
+// 	path := "tests/templates/validate-fail/"
+// 	err := validateKyvernoPolicies(path)
+// 	if err == nil {
+// 		t.Fatalf("Expected validation to fail for manifest %s, but it passed", path)
+// 	}
+// }
```

Comment:
why is this commented out?

Reply:
not sure how this crept in there. i was testing with these changes. uncommented now.